OpenRA.Mods.RA/Traits/Mine.cs

```diff
@@ -52,6 +52,9 @@ void INotifyCrushed.OnCrush(Actor self, Actor crusher, HashSet<string> crushClas
 
 		bool ICrushable.CrushableBy(Actor self, Actor crusher, HashSet<string> crushClasses)
 		{
+			if (!crusher.Info.HasTraitInfo<MineImmuneInfo>() && self.Owner.Stances[crusher.Owner] == Stance.Ally && info.AvoidFriendly)
```

Comment:
Depends on wether we want friendly units to be able to enter the cell containing a mine or not.
However, this should use another bool than `AvoidFriendly` (maybe `BlockFriendly`?) if we want it.